src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -203,24 +201,26 @@ private static void analyzeRepos(String outputPath, List<RepoConfiguration> conf
             } catch (NoAuthorsWithCommitsFoundException nafe) {
                 logger.log(Level.WARNING, String.format(MESSAGE_NO_AUTHORS_WITH_COMMITS_FOUND,
                         configToAnalyze.getLocation(), configToAnalyze.getBranch()));
-                generateEmptyRepoReport(repoReportDirectory.toString(), Author.NAME_NO_AUTHOR_WITH_COMMITS_FOUND);
+                generatedFiles.addAll(
+                    generateEmptyRepoReport(repoReportDirectory.toString(), Author.NAME_NO_AUTHOR_WITH_COMMITS_FOUND));
             }
         }
+        return generatedFiles;
     }
 
     /**
      * Analyzes repo specified by {@code config} and generates the report.
      */
-    private static void analyzeRepo(
+    private static List<Path> analyzeRepo(
```

Comment:
Same here

Reply:
Done. I've indicated the return value in the Javadoc.